src/scripts/Controller.js

```diff
@@ -640,7 +678,7 @@ export default class SitemapController {
 	editSitemapMetadata(button) {
 		this.setActiveNavigationButton('sitemap-edit-metadata');
 
-		let sitemap = this.state.currentSitemap.clone();
+		let sitemap = this.state.currentSitemap;
```

Comment:
There was a bit of misunderstanding here. I've introduced this change in 2e3a814 and pushed it to `dev` branch, and you placed your work on top of it. Then you made this pull request from your branch to `master`, so my commit was included in diff. Alex has changed the target branch, but the conversation was left unresolved, and you decided to revert this change. So, I think now you have to get it back)